.github/workflows/prebuild.yml

```diff
@@ -3,19 +3,20 @@ name: "Prebuild checks"
 on: [pull_request]
 
 jobs:
-
   style_check:
+    name: Style check
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2.3.4
       - uses: cachix/install-nix-action@v13
         with:
           nix_path: nixpkgs=channel:nixos-unstable
-        name: "Run script"
+        name: "Install poetry and run style check"
```

Comment:
I would drop the "Install poetry" as it is a prerequisite not something we test per-se.

Reply:
Ok removed the install poetry and left only the description "Run style check" or other checks.